### PR TITLE
Update to Circle2.0

### PR DESCRIPTION
**Pull request by jasonwc**
**Summary**

This uses CircleCI's [config translation](https://circleci.com/docs/2.0/config-translation/) endpoint to bump brainstem-redux to Circle2.0

**Test plan**

Ensure that this build correctly, and we're g2g.

### CODE REVIEW

**Comment by juanca** [comment]
Looks like it stalled in `yarn install`. SSH into the box now

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+jobs:
+  build:
+    working_directory: ~/mavenlink/brainstem-redux
+    parallelism: 1
+    shell: /bin/bash --login
+    environment:
+      CIRCLE_ARTIFACTS: /tmp/circleci-artifacts
+      CIRCLE_TEST_REPORTS: /tmp/circleci-test-results
+    docker:
+    - image: circleci/build-image:ubuntu-14.04-XXL-upstart-1189-5614f37
```
Comment (juanca) [inline .circleci/config.yml]: I'm going to tweak this. Building without cache is ridiculously expensive.

**Comment by juanca** [comment]
Changed the base image.

Rebuilding without cache took 5 minutes to spin up... Now it takes 1 second. Hooray!

**Comment by jasonwc** [comment]
Sweet! Good call on the base image.

https://youtu.be/Dr0Qlftwa4c?t=16s

**Pull request merged**